docs/docs/Enterprise/audit_logs.md

```diff
@@ -92,3 +92,50 @@ The dropdown will display all the apps associated with your account. Select an a
 | resource_type | Indicates the type of the [resources](#select-resources) involved in the logged event. |
 | user_id | Each user account in ToolJet has a unique ID associated with it, which is recorded when an event occurs. |
 
+### Log file
+
+The log file can be created by specifying the path in the [environment variables](/docs/setup/env-vars) . The file will contain all the data from audit logs. The file will be updated every time a new audit log is created.
+
```

Comment:
"The file will be updated every time a new audit log is created." -> since file is updating daily, it will be better we will say something every day new log file generation,